playbooks/zuul/templates/group_vars/gitea.yaml.j2

```diff
@@ -3,7 +3,3 @@ gitea_secret_key: "dummy-secret-key"
 gitea_db_type: "sqlite3"
 gitea_cert: "gitea"
 
-gitea_smtp_from_address: "gitea@smtp.local"
```

Comment:
and here they can be also becoming gitea_mailer_xxx

Reply:
removed as in check job mailer part is disabled by default